ui/components/ui/contract-token-values/contract-token-values.js

```diff
@@ -65,8 +65,9 @@ export default function ContractTokenValues({
       </Box>
       <Box className="contract-token-values__block-explorer">
         <Tooltip position="top" title={t('openInBlockExplorer')}>
-          <Button
-            type="link"
+          <ButtonIcon
+            iconName={ICON_NAMES.EXPORT}
+            color={COLORS.ICON_MUTED}
```

Comment:
Since this was a link, I think we should be using [ButtonLink](https://metamask.github.io/metamask-storybook/?path=/story/components-componentlibrary-buttonlink--default-story) here as a wrapper and use ButtonIcon for the IconBlockExplorer only. Wdyt?

Reply:
Thoughts here @georgewrmarshall ?

Reply:
Good spotting @NidhiKJha! I think the designer probably intended for these buttons to have less visual hierarchy by using `colors.iconMuted` here. This doesn't technically break any design system rules but I think this would probably effect accessibility somewhat. My recommendation would be to still use `ButtonIcon` but update the color to use `color={Color.primaryDefault}`

<img width="602" alt="Screenshot 2023-02-05 at 8 14 39 PM" src="https://user-images.githubusercontent.com/8112138/216882501-306e1c12-74b5-4c14-b788-4eecbff90018.png">

Reply:
Sweet! Thanks George 😄